### ISSUE TITLE
Use `"$@"` instead of `$@`

### ISSUE THREAD

**Issue by kyoh86** [kyoh86/vim-editerm#10]
shellcheck

```
if [[ -n "${VIM_TERMINAL}" ]]; then
  function vim() {
    ${EDITOR} "$@"
  }
fi
```

ref: https://vim-jp.slack.com/archives/CLKR04BEF/p1566111297282100
ref: https://qiita.com/Ping/items/57fd75465dfada76e633

**Comment by tsuyoshicho** [comment]
And then
`$EDITOR` to ` ${EDITOR}` change do no problem?

**Comment by kyoh86** [comment]
it's fixed in 5648a6f7940861c10a63c17aacfeba3fcba339c8
